### PR TITLE
#20506 Settings "Group->Show category" and "Group->Font size" are not…

### PR DESCRIPTION
**Pull request by vtkalek**
… working

#20421 Add instruction to Sunburst chart (with pictures)
#20477 Add colors selection in settings pane
#14627 Add an ability to support decimal formatting

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.1.2
```
Comment (ignatvilesov) [inline CHANGELOG.md]: Let's use `1.2.0` due to fact that we are going to incorporate new features.

File: capabilities.json [hunk 2]
```diff
@@ -80,6 +80,28 @@
         }
       }
     },
+    "tooltip": {
+      "displayName": "Tooltip",
+      "displayNameKey": "Visual_Tooltip",
+      "properties": {
+      "displayUnits": {
```
Comment (ignatvilesov) [inline capabilities.json]: Please format this file using VS Code.
This line should have extra 4 spaces

File: src/SunburstVisual.ts [hunk 3]
```diff
@@ -323,13 +347,16 @@ module powerbi.extensibility.visual {
                             .attr("id", "sliceLabel_" + i);
                     }
                 });
+                let labelShift: number = 26;
```
Comment (uve) [inline src/SunburstVisual.ts]: Move it to class static members

File: src/SunburstVisual.ts [hunk 4]
```diff
@@ -323,13 +347,16 @@ module powerbi.extensibility.visual {
...
                 self.main.selectAll(this.appCssConstants.sliceLabel.selectorName)
                     .data<SunburstSlice>(<any>partition.nodes)
                     .enter()
                     .append("text")
                     .classed(this.appCssConstants.sliceLabel.className, true)
                     // font size + slice padding
-                    .attr("dy", 18)
+                    .attr("dy", (d: SunburstSlice) => {
```
Comment (uve) [inline src/SunburstVisual.ts]: Why do you need **d: SunburstSlicer** object here?

File: test/visualTest.ts [hunk 5]
```diff
@@ -202,17 +202,16 @@ namespace powerbi.extensibility.visual.test {
                     DefaultWaitForRender);
             });
 
-            it("data labels should be hidden by default", (done: DoneFn) => {
+            it("data labels should not be hidden by default", (done: DoneFn) => {
```
Comment (uve) [inline test/visualTest.ts]: Add more unit-tests for:
+* "Group->Show category" and "Group->Font size" were not working
 +* Add colors selection in settings panel
 +* Add an ability to show values of the inner ring
 +* Add an ability to support decimal formatting

Comment (vtkalek) [reply test/visualTest.ts]: Added 2 tests

**Pull request merged**